packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -200,11 +202,15 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
                                 data-ng-model="node.ahvIp"
                                 data-ng-model-options="{ debounce: 500 }"
                                 data-ng-pattern="$ctrl.IPV4_REGEX"
+                                data-unique-ip-validator="$ctrl.getIpsList()"
```

Comment:
as i seen, we call this list in may others places.
I think we must consider to initialize a `this.ipsList = this.getIpsList()` into the controller.

Reply:
This list is dynamic and needs to be generated on change of all the IP fields. So the above suggestion cannot be implemented.